cli/command/swarm/ca.go

```diff
@@ -61,6 +61,10 @@ func runRotateCA(dockerCli command.Cli, flags *pflag.FlagSet, opts caOptions) er
 	}
 
 	if !opts.rotate {
+		if flags.Changed(flagCACert) || flags.Changed(flagCAKey) || flags.Changed(flagCertExpiry) {
+			fmt.Fprintln(dockerCli.Out(), "Warning: because the `--rotate` flag was not provided, all other flags will be ignored")
+		}
```

Comment:
Any reason not to make this an error instead of a warning? 

If this remains a warning it should be on `stderr` (`dockerCli.Err()`). 

If it remains a warning maybe this could be split out into a separate function and we can print a warning for each flag? "all other flags" isn't really accurate because `-d` and `-q` are allowed, and we could always add other flags later.

Something like 

"WARNING: Flag --%s ignored because, you must --rotate to update the CA"